### PR TITLE
[DOC] add "maintainers" column to estimator overview

### PR DESCRIPTION
**Pull request by fkiraly**
Adds a "maintainers" column to the estimator overview, to facilitate pinging of maintainers

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -360,7 +362,8 @@ def _does_not_start_with_underscore(input_string):
             + "</a>"
         )
 
-        record = pd.DataFrame([modname, algorithm_type, author_info], index=COLNAMES).T
+        record_cells = [modname, algorithm_type, author_info, maintainer_info]
+        record = pd.DataFrame(record_cells, index=COLNAMES).T
```
Comment (yarnabrina) [inline docs/source/conf.py:366]: This line does look a bit weird. If I understand this correctly, may be you can use this type of syntax?

```pycon
>>> 
>>> import pandas
>>> 
>>> row_names = ["r1", "r2"]
>>> column_names = ["c1", "c2", "c3"]
>>> 
>>> cell_values = [["r1c1", "r1c2", "r1c3"], ["r2c1", "r2c2", "r2c3"]]
>>> 
>>> table = pandas.DataFrame(data=cell_values, index=row_names, columns=column_names)
>>> 
>>> table
      c1    c2    c3
r1  r1c1  r1c2  r1c3
r2  r2c1  r2c2  r2c3
>>> 
```

So, you can keep `records` object as list of lists of row data, and create the dataframe once at the end.

**Review by yarnabrina — CHANGES_REQUESTED** [review]

File: docs/source/conf.py (hunk 1 above)
Comment (fkiraly) [reply docs/source/conf.py:366]: I see - sure.
It was like that when I found it already.

btw, do you know how to make search and subset in html? Or sorting, as here:
https://en.wikipedia.org/wiki/Help:Sortable_tables#Sorting_buttons_in_a_separate_row

Currently, the "search box" does not work

Comment (yarnabrina) [reply docs/source/conf.py:366]: No, sorry. I just know html-css-js is a thing, and that's where my knowledge ends abruptly ;(

But previously @Abhishektharu pointed out this, if you can use it in some way.

Comment (fkiraly) [reply docs/source/conf.py:366]: > But previously @Abhishektharu pointed out this, if you can use it in some way.

did you mea to add a link?

Comment (yarnabrina) [reply docs/source/conf.py:366]: I linked to the [original post](https://github.com/sktime/sktime/issues/5495#issuecomment-1786648195), don't know why you can't see.

> I found that the feature (search the table ) worked in v0.20.0 and previous but v0.20.1 and later caused problem. In console i found that (Uncaught ReferenceError: $ is not defined at estimator_overview.html:1992:9) this mainly happens when we don't include jQuery is not properly loaded or initialized in HTML and it is missing after v0.20.0. But it was included in v0.20.0. Screenshot 2023-10-31 131856 **probably including jQuery in head section of html will solve the problem** Screenshot 2023-10-31 132037

Comment (fkiraly) [reply docs/source/conf.py:366]: > I linked to the [original post](https://github.com/sktime/sktime/issues/5495#issuecomment-1786648195), don't know why you can't see.

not in your original post, you added the link by edit

File: docs/source/conf.py [hunk 2]
```diff
@@ -360,7 +362,8 @@ def _does_not_start_with_underscore(input_string):
             + "</a>"
         )
 
-        record = pd.DataFrame([modname, algorithm_type, author_info], index=COLNAMES).T
+        record_cells = [[modname, algorithm_type, author_info, maintainer_info]]
+        record = pd.DataFrame(data=record_cells, columns=COLNAMES)
         records += [record]
 
     df = pd.concat(records, ignore_index=True)
```
Comment (yarnabrina) [inline docs/source/conf.py:369]: I think I didn't explain myself correctly. This is what I meant.

```suggestion
        records.append([modname, algorithm_type, author_info, maintainer_info])

    df = pd.DataFrame(records, columns=COLNAMES)
```

**Review by yarnabrina — CHANGES_REQUESTED** [review]

**Review by yarnabrina — APPROVED** [review]
It's working after the suggestion as well.

image

**Pull request merged**